### PR TITLE
Fix dummy real attributes

### PR DESCRIPTION
**Pull request by lochel**
see [ticket:4165](https://trac.openmodelica.org/OpenModelica/ticket/4165)

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-12-02_16-28-47](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3495/console).

File: SimulationRuntime/c/simulation_data.h [hunk 1]
```diff
@@ -47,7 +47,7 @@
 #define omc_dummyVarInfo {-1,-1,"","",omc_dummyFileInfo}
 #define omc_dummyEquationInfo {-1,0,"",-1,NULL}
 #define omc_dummyFunctionInfo {-1,"",omc_dummyFileInfo}
-#define omc_dummyRealAttribute {NULL,NULL,DBL_MAX,-DBL_MAX,0,0,1.0,0,0.0}
+#define omc_dummyRealAttribute {NULL,NULL,DBL_MAX,-DBL_MAX,0,0,1.0,0.0}
```
Comment (hkiel) [inline SimulationRuntime/c/simulation_data.h]: Are you sure that `modelica_real min; /* = -Inf */` is `DBL_MAX` and
`modelica_real max; /* = +Inf */` is `-DBL_MAX`?

Comment (lochel) [inline SimulationRuntime/c/simulation_data.h]: Oops!

Comment (sjoelund) [inline SimulationRuntime/c/simulation_data.h]: It is correct. DBL_MIN is a small number. -DBL_MAX is a very large, negative number.

Comment (lochel) [inline SimulationRuntime/c/simulation_data.h]: This is just about the flipped arguments.

**Comment by hkiel** [comment]
What is "min" and "max"? Smallest and largest postive number?
Then it should be min=DBL_**MIN** and max=DBL_**MAX**.

**Comment by lochel** [comment]
The dummy attributes are not used. However, the min and max values are the real variable attributes from Modelica. So min should be –DBL_MAX and max should be +DBL_MAX.

**Pull request closed without merging**